providers/marketo/connnector.go

```diff
@@ -10,9 +10,10 @@ import (
 )
 
 type Connector struct {
-	BaseURL string
-	Client  *common.JSONHTTPClient
-	Module  string
+	BaseURL    string
+	Client     *common.JSONHTTPClient
+	ModuleID   paramsbuilder.APIModuleID
```

Comment:
This doesn't feel easy to remember - can you find a way to just have a `Module` concept which gives us access to the `Id` & the `Path`?